esphome/components/api/api_connection.cpp

```diff
@@ -104,6 +104,7 @@ void APIConnection::loop() {
       ESP_LOGW(TAG, "%s didn't respond to ping request in time. Disconnecting...", this->client_info_.c_str());
     }
   } else if (now - this->last_traffic_ > keepalive) {
+    ESP_LOGD(TAG, "Sending keepalive PING...");
```

Comment:
```suggestion
    ESP_LOGVV(TAG, "Sending keepalive PING...");
```

(don't want to spam logs)